modules/4337/certora/harnesses/Account.sol

```diff
@@ -53,6 +53,78 @@ contract Account is Safe {
     function getValidUntilTimestamp(bytes calldata sigs) external pure returns (uint48) {
         return uint48(bytes6(sigs[6:12]));
     }
+
+    /*
+    Actual Signature:
+    "0x" +
+    "0000000000000000000000000000000000000000000000000000000000000001 00000000000000000000000000000000000000000000000000000000000000c3 00" +     // encoded EIP-1271 signature 1
```

Comment:
1. I don't understand the purpose of having all different signature types here with actual values. I think just explaining in short why all of the signatures will have same hash and then linking to docs https://docs.safe.global/advanced/smart-account-signatures should be sufficient.

2. we are writing solidity code to verify some other solidity code. I would be careful in doing so as we are not verifying this solidity code here and preferably avoid writing additional code.  I understand that this code is needed to verify other code but, we are then relying on correctness of this code. We have done similar things in previous FV setups by having harnessed contracts but it was limited to reading contract state and simpler code snippets. Is there any alternative approach?

Reply:
> Ironically, the doc has some wrong values. Made a PR to update that: https://github.com/safe-global/safe-docs/pull/527 Maybe we can move the details to a doc and add that link here? Wdyt?

The docs would be fixed soon, right? I think linking to official docs would be the nicest way to do it rather than having another doc somewhere else with similar repeated info.

Reply:
But the docs don't show how the signature could be manipulated (which is good 😅 ) and still make the signature check in Safe pass directly, though will be caught within the 4337 Module.

Reply:
I agree, I find this comment a bit excessive TBH.

> we are writing solidity code to verify some other solidity code

Agree - this is a weakness to using a harness function. I think the issue is that expressing this in CVL is quite hard.

Reply:
> I agree, I find this comment a bit excessive TBH.

Do you recommend removing it altogether, or moving it? Or just write the same in fewer words rather than the example?

Reply:
I have written a shorter comment with a smaller example and a few lines of explanation.